package.json

```diff
@@ -19,9 +19,10 @@
   ],
   "scripts": {
     "build": "npm run copy:assets && tsc 1>&2",
-    "build:debug": "npm run build && run-script-os",
-    "build:debug:nix": "./syncToTestApp.sh",
-    "build:debug:windows": "powershell ./syncToTestApp.ps1",
+    "build:debug": "npm run build && npm run synctotestapp",
```

Comment:
What was the reason for this set of changes? (I don't mind them being here, I just don't understand why they are here.

Reply:
It allows us to run synctotestapp from other projects. For example, I'm using it in mobile-samples and locally in Synchro Field.